klasses/models_test.py

```diff
@@ -171,3 +193,29 @@ def test_total_due_by_next_deadline():
         bootcamp_run.total_due_by_next_deadline
         == installment_past.amount + installment_next.amount
     )
+
+
+def test_bootcamp_run_certificate(bootcamp_run, user):
+    """
+    test bootcamp run certificates
+    """
+
+    certificate = BootcampRunCertificateFactory.create(
+        user=user, bootcamp_run=bootcamp_run
+    )
+
+    assert certificate.is_revoked is False
+    assert certificate.start_end_dates == (
+        bootcamp_run.start_date,
+        bootcamp_run.end_date,
+    )
+    assert certificate.link == "/certificate/{}/".format(str(certificate.uuid))
+    assert str(
+        certificate
+    ) == 'BootcampRunCertificate for user={user}, run={bootcamp_run} ({uuid})"'.format(
```

Comment:
```suggestion
    ) == 'BootcampRunCertificate for user={user}, run={bootcamp_run} ({uuid})'.format(
```
Also remove that double-quotes from `BootcampRunCertificate.__str__` method

Reply:
that `"` is also placed in the view so I kept it the same there. I did not notice that. By the way, I have removed it in both 
https://github.com/mitodl/bootcamp-ecommerce/blob/3d1df795f9a1360748e6f31b34e94c1723143b65/klasses/models.py#L343